build-system/tasks/e2e/index.js

```diff
@@ -80,20 +80,28 @@ function createMocha_() {
   let reporter;
   if (argv.testnames || argv.watch) {
     reporter = '';
-  } else if (argv.report) {
+  } else if (argv.report || isCircleciBuild()) {
     reporter = ciReporter;
   } else {
     reporter = dotsReporter;
   }
 
-  return new Mocha({
+  const options = {
```

Comment:
Can this stay the same but with `reporterOptions: isCiBuild() ?? {...}`

Reply:
That should be fine but `reporterOptions` will always exist this way.